pkg/graph/resolvers/itgovactions/lcidactions/unretire_lcid.go

```diff
@@ -0,0 +1,17 @@
+package lcidactions
+
+import (
+	"github.com/cms-enterprise/easi-app/pkg/models"
+)
+
+// GetUnretireLCIDAction is a method to create an action record with all the relevant data fields for an Unretire LCID action.
+func GetUnretireLCIDAction(
+	intake models.SystemIntake,
+	userInfo models.UserInfo,
+) models.Action {
+	// pass in nil for most parameters - only change is to set the retirement date to nil
+	action := getBaseLCIDAction(intake, nil, nil, nil, nil, userInfo, nil)
```

Comment:
i see a lot of optional parameters get passed in, would it make sense to either add an optional param for an action type or create a struct with optional fields and pass just one thing in?

Reply:
Might be worth a `TODO` or revisiting, but probably not worth it right now if we're alright just leaving the comment here.